src/Groups/group_characters.jl

```diff
@@ -261,31 +302,44 @@ function matrix_of_strings(tbl::GAPGroupCharacterTable; alphabet::String = "")
       elseif val.c == 1
         m[i, j] = string(val.data)
       elseif alphabet != ""
-        # write irrationalities using symbols
+        # write irrationalities using symbolic names
         pos = findnext(x -> x[1] == val, legend, 1)
         if pos != nothing
           m[i,j] = legend[pos][2]
         else
           pos = findnext(x -> x[1] == -val, legend, 1)
           if pos != nothing
+            # The negative of a known name is shown relative to that name.
             m[i,j] = "-" * legend[pos][2]
           else
             name, state = iterate(iter, state)
-            push!(legend, val => name)
+            disp = as_sum_of_roots(val.data, root)
+            push!(legend, (val, name, disp))
             valbar = complex_conjugate(val)
             if valbar != val
-              push!(legend, valbar => "\\overline{"*name*"}")
-#TODO: represent the unique Galois conjugate of `A` different from `A` by `A*`,
-#      and show both `A` and `A*` in the footer;
-#      for elements in quadratic fields, show also an expression in terms of
+              # The complex conjugate of a known name is shown relative to
+              # that name.
+              disp = as_sum_of_roots(valbar.data, root)
+              push!(legend, (valbar, "\\overline{"*name*"}", disp))
```

Comment:
I am told in idiomatic Julia, string interpolation is preferred over string concatenation
```suggestion
              push!(legend, (valbar, "\\overline{$name}", disp))
```